README.md

```diff
@@ -44,17 +44,17 @@ This is the official Komodo sourcecode repository based on https://github.com/jl
 - Pegged Assets - Chains that maintain a peg to fiat currencies  
 - Peerchains - Scalability solution where sibling chains form a network of blockchains  
 - More in depth covered [here](https://docs.google.com/document/d/1AbhWrtagu4vYdkl-vsWz-HSNyNvK-W-ZasHCqe7CZy0)  
```

Comment:
This link is not public. Suggest removing it for now - we can import it into the docs site and add link back again later.

Reply:
Removed in https://github.com/KomodoPlatform/komodo/pull/647/commits/eb3b4afff7550300df635a85bbefa89c4b76cafa .